YuriiDatsiuk/HW5/5.3.py

```diff
@@ -0,0 +1,10 @@
+number = int(input('Enter number to calculate her factorial '))
```

Comment:
look at this implementation
number = int(input('Enter number to calculate its factorial: '))

if number < 0:
    print("Please enter a non-negative integer.")
else:
    result = 1
    for i in range(1, number + 1):
        result *= i

    print(f"The factorial of {number} is: {result}")